doc/ref_impl/cntt-ri2/chapters/chapter04.md

```diff
@@ -44,34 +45,9 @@ IDF includes information about network information required by the installer. Al
-
-`./deploy.sh -o <OStype>-p file:///<pdf.yaml> -i file:///<idf.yaml> -l provision`
-
-Currently, Ubuntu 18.04 and CentOS 7.8 are supported (default Ubuntu 18.04). Support for other operating systems can be added as well depending on the requirements.
-
-After the hosts have been provisioned successfully, one can set up host networking for Kubernetes and run Kubernetes provisioning tooling from CNF Testbed or Intel’s BMRA playbooks to configure and install k8s and other plugins (Refer 4.3.2)
-
-<a name="4.3.2"></a>
-### 4.3.2 Kubernetes Provisioning
-
-Early efforts to provision Kubernetes are based on existing tools such as [Bare Metal Reference Architecture (BMRA)](https://builders.intel.com/docs/networkbuilders/container-bare-metal-for-2nd-generation-intel-xeon-scalable-processor.pdf), using scripts available on [Github](https://github.com/intel/container-experience-kits), and the CNCF initiative [CNF Testbed](https://github.com/cncf/cnf-testbed). 
-
-Requirements are based on the [Kubernetes Based Reference Architecture (RA2)](../../../ref_arch/kubernetes), with initial focus on hardware and functionality related to CPU and networking. While this will not be sufficient to satisfy all the requirements listed in RA2, it will serve as a development platform for further feature integration, verification and ideally testing related to the [Kubernetes Based Reference Conformance (RC2)](../../../ref_cert/RC2).
-
-The Kubernetes provisioning follows the approach described in [Chapter 1](chapter01.md#13-reference-implementation-approach) of this RI. Steps for preparing the BMRA environment can be found here: [CNTT RI2 Confluence page](https://wiki.lfnetworking.org/display/LN/Kubernetes+Bare-Metal+Install+and+Configuration). 
-
-Make sure all servers are reachable from the jumphost through SSH. If necessary, create a key and push it to the master and worker nodes
-```
-$ ssh-keygen -t rsa -b 4096
-$ ssh-copy-id -i <path to public key> <user>@<host>
-```
-
-The following paragraphs will use v1.4.1 of the BMRA, and focus on configuration options that are relevant in terms of the RA2 and RI2.
+Additionally, modify the BMRA config by first updating `sw_config/bmra/inventory.ini` with the correct servers and functionality planned for each node. For the high availability requirement at least 3 nodes should be running as master with etcd enabled, but only a single master (and worker) is required to deploy the cluster.
 
-Start by updating `inventory.ini` with the correct servers and functionality planned for each node. For the high availability requirement at least 3 nodes should be running as master with etcd enabled, but only a single master (and worker) is required to deploy the cluster.
-
-Next, update parts of `group_vars/all.yml`:
+Next, update parts of `sw_config/bmra/all.yml`:
```

Comment:
I recommend to clarify that these configuration options exist, but as part of the official RI-2 they are set to specific values according to RA-2 requirements. So, they should actually not be changed if the goal is to deploy a reference platform. The values can be changed for debugging and development purposes, though.

Reply:
Done. In future we could also consider moving it into Kuberef deployment guide.